src/plugins/intel_gpu/src/plugin/infer_request.cpp

```diff
@@ -25,6 +25,18 @@ const char str_device_output_unsupported_blob[] = "Device output is of an unsupp
 const char str_input_not_allocated[] = "Input data was not allocated.";
 const char str_output_not_allocated[] = "Output data was not allocated.";
 
+static bool can_use_usm_host(const cldnn::engine& engine) {
+    auto can_use_usm = engine.use_unified_shared_memory();
+
+    // WA: Disable USM host memory for infer request`s tensors for PVC as
+    // it has performance issues in case of host <-> device data transfers inside kernels
+    auto supported_simd_sizes = engine.get_device_info().supported_simd_sizes;
+    if (std::find(supported_simd_sizes.begin(), supported_simd_sizes.end(), 8) == supported_simd_sizes.end())
```

Comment:
Why not checking `gfx_ver` instead? Do we still have version parsing issues?

Reply:
@vladimir-paramuzov, how fixed version is supposed to look like? Something like "12.8.0"?

Reply:
Yes, 12.8.x:
https://github.com/intel/compute-runtime/blob/master/opencl/source/xe_hpc_core/cl_gfx_core_helper_xe_hpc_core.cpp#L29